src/web/mjs/connectors/Manhuaes.mjs

```diff
@@ -0,0 +1,59 @@
+import Connector from '../engine/Connector.mjs';
+import Manga from '../engine/Manga.mjs';
+
+export default class ManhuaES extends Connector {
+
+    constructor() {
+        super();
+        super.id = 'manhuaes';
+        super.label = 'Manhua ES';
+        this.tags = [ 'manga', 'English' ]; //should I add 'Webtoon' tag as well?
+        this.url = 'https://manhuaes.com';
+    }
+
+    async _getMangaFromURI(uri) {
+        let request = new Request(uri, this.requestOptions);
+        let data = await this.fetchDOM(request, 'h1.title-detail');
+        let id = uri.pathname;
+        let title = data[0].textContent;
+        return new Manga(this, id, title);
+    }
+
+    async _getMangas() {
+        let request = new Request(new URL('/category-comics/manga/', this.url), this.requestOptions);
+        let pages = await this.fetchDOM(request, 'ul.pagination li:nth-last-child(1) > a');
+        pages = Number(pages[0].pathname.match(/\/(\d+)\/$/)[1]);
```

Comment:
It seems to be easier to use `Number(pages[0].text)` instead of extracting the number via regex from the `href` attribute